gap/oper.gi

```diff
@@ -1239,6 +1239,67 @@ function(digraph, u, v)
   return DIGRAPH_PATH(OutNeighbours(digraph), u, v);
 end);
 
+# DigraphShortestPath
+
+InstallMethod(DigraphShortestPath, "for a digraph and two pos ints",
+[IsDigraph, IsPosInt, IsPosInt],
+function(digraph, u, v)
+  local current, next, parent, distance, falselist, verts, nbs, n, a, b, i, path;
+
+  verts    := DigraphVertices(digraph);
+  nbs      := OutNeighbors(digraph);
+  distance := [];
+
+  if u = v and v in nbs[v] then  # Considers the trivial path from v to v
```

Comment:
A loop is not a trivial path. Therefore please change this comment, to something like `# is u = v, and there is a loop at v?`